actioncable/lib/action_cable/channel/broadcasting.rb

```diff
@@ -7,22 +7,26 @@ module Channel
     module Broadcasting
       extend ActiveSupport::Concern
 
-      delegate :broadcasting_for, to: :class
+      delegate :broadcasting_for, :broadcast_to, to: :class
 
       module ClassMethods
         # Broadcast a hash to a unique broadcasting for this <tt>model</tt> in this channel.
         def broadcast_to(model, message)
-          ActionCable.server.broadcast(broadcasting_for([ channel_name, model ]), message)
+          ActionCable.server.broadcast(broadcasting_for(model), message)
         end
 
         def broadcasting_for(model) #:nodoc:
+          stringify_broadcasting([ channel_name, model ])
+        end
+
+        def stringify_broadcasting(object)
```

Comment:
This method should be `#:nodoc`ed or made private. It doesn’t seem to be used elsewhere, so preferably the latter.

Reply:
Let’s also rename to serialize_broadcasting. Stringify sounds JS-like.

Reply:
Sure.

> Stringify sounds JS-like.

Yep) I didn't like it either)

Thanks!